model_compression_toolkit/trainable_infrastructure/keras/config_serialization.py

```diff
@@ -17,12 +17,17 @@
 from typing import Any, Union
 from enum import Enum
 
+import numpy as np
+
 from model_compression_toolkit.target_platform_capabilities.target_platform import QuantizationMethod
 from model_compression_toolkit.trainable_infrastructure.common.trainable_quantizer_config import \
     TrainableQuantizerActivationConfig, TrainableQuantizerWeightsConfig
 from mct_quantizers.common import constants as C
 
 
+CONFIG = "config"
```

Comment:
maybe put these in our constants file? (and then use them in relevant exporter tests as well)

Reply:
I thought about that but is very very specific to here